protocol/synthetix/subgraph/base-sepolia-andromeda/collateralDefinition.ts

```diff
@@ -0,0 +1,39 @@
+import { Address, BigInt } from '@graphprotocol/graph-ts';
+
+// Initialize a Token Definition with the attributes
+export class TokenDefinition {
+  address: Address;
+  symbol: string;
+  name: string;
+  decimals: BigInt;
+
+  // Get all tokens with a static defintion
+  static getStaticDefinitions(): Array<TokenDefinition> {
```

Comment:
Why do we need static definitions for tokens? This instantly adds the hard dependency on graph for any new collateral additions. We should pull collateral configurations in the ui and do the matching in the ui and not in subgraph

Reply:
Yes it does. 
We are pulling them in the UI now already, this simplifies things for loading across diff networks.

Reply:
It does not. Because it means that we now lock in multiple things in the chain of updates. After deployment we would need to redeploy subgraph, do full indexing and then things will start working in ui. If we just pull data from contracts then after deployment update ui will pick things up automatically

Reply:
I am 100% against this change.